### PR TITLE
Add Palindromic Number in Typescript

### LINKED ISSUE #3386: Add Palindromic Number in Typescript
To request a new code snippet, please fill out the following:

Project name: Palindromic Number
Project link: https://sampleprograms.io/projects/palindromic-number
Language: Typescript

> The project link must be a link to a project on the [Sample Programs Website project list][sample-programs-project-list].
> If you would like to add a new project to the Sample Programs Website,
> please first make a pull request to the [Sample Programs Website][sample-programs-website].

[sample-programs-website]: https://github.com/TheRenegadeCoder/sample-programs-website
[sample-programs-project-list]: https://sampleprograms.io/projects/

### PR DESCRIPTION
**Pull request by MeetThakur**
Congrats on taking the first step to contributing to the Sample Programs repository maintained by [The Renegade Coder][renegade-coder]! 
For simplicity, please make sure that your pull request includes one and only one contribution.

Please fill _one_ of the sections below as applicable.
Please also add any other relevant information to the Notes section at the bottom.
You may delete or just ignore any other sections.
For more information please refer to our [contributing documentation][contributing]

## I Am Adding a New Code Snippet in an Existing Language

- [x] I fixed #3386 
- [x] I named the pull request using `Add {PROJECT} in {LANGUAGE}` format
  

## Other Notes

Add notes here...

[renegade-coder]: https://therenegadecoder.com/
[contributing-plagiarism]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#plagiarism
[contributing-new-project]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#requirements-for-a-new-project
[contributing-new-language]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#requirements-for-a-new-language
[contributing-readme]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#create-readmes
[contributing-tests-in-detail]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#tests-in-detail
[contributing]: ../CONTRIBUTING.md
[sample-programs-project-list]: https://sampleprograms.io/projects/
[contributing-modifications]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#modifying-existing-code-snippets

### CODE REVIEW

**Review by rzuckerm — CHANGES_REQUESTED** [review]
Thanks for your submission. Please use command-line arguments instead of `prompt`. Unfortunately, there is no existing sample in Typescript that deals with numbers. However, there is the [Reverse String in Typescript sample](https://github.com/TheRenegadeCoder/sample-programs/blob/main/archive/t/typescript/reverse-string.ts#L1) that deals with command-line arguments.

**Comment by MeetThakur** [comment]
I have updated the code accordindly can you check now.

File: archive/t/typescript/Palindromic-Number.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+} else {
+    const input = process.argv[2];
+    const numberToCheck = parseFloat(input);
+
+    if (!isNaN(numberToCheck) && Number.isInteger(numberToCheck) && numberToCheck >= 0) {
+        if (isPalindrome(numberToCheck)) {
+            console.log(`${numberToCheck} is a palindrome.`);
+        } else {
+            console.log(`${numberToCheck} is not a palindrome.`);
+        }
```
Comment (rzuckerm) [inline archive/t/typescript/Palindromic-Number.ts:18]: Sorry, I missed the first time around. This is supposed to output either `"true"` or `"false"`:
```suggestion
        if (isPalindrome(numberToCheck)) {
            console.log("true");
        } else {
            console.log("false");
        }
```

**Review by rzuckerm — CHANGES_REQUESTED** [review]

File: archive/t/typescript/Palindromic-Number.ts (hunk 1 above)
Comment (MeetThakur) [reply archive/t/typescript/Palindromic-Number.ts]: sorry my bad I did'nt see that too.

**Comment by rzuckerm** [comment]
@MeetThakur Looks good! If the build passes, I'll go ahead and merge this.

**Review by rzuckerm — APPROVED** [review]

**Comment by jrg94** [comment]
Nice work! Thanks to the both of you for taking the time to get this working.

**Pull request merged**